source/MaterialXView/Main.cpp

```diff
@@ -300,19 +296,26 @@ int main(int argc, char* const argv[])
         viewer->setBakeHeight(bakeHeight);
         viewer->setBakeFilename(bakeFilename);
         viewer->initialize();
-        if (!bakeFilename.empty()) 
+
+        bool drawOffscreen = !bakeFilename.empty() || !captureFilename.empty();
```

Comment:
The pathing change above looks great, but I'm less clear on the motivation for this change.  If we're baking textures to disk, why would the viewer need to draw at all?  And why would we need to explicitly set the visibility of the viewer to false, when it defaults to that setting?

Reply:
I've fixed this up. Only the capture needs a forced redraw as the viewport is not set to be visible before performing the capture.